docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+upgrade of agents considerably easier.
+
+On Kubernetes, deploy {agent} as a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[DaemonSet]
+to ensure that there is a running instance on each node of the cluster.
+These instances are used to retrieve metrics from the host, such as system metrics, container stats,
+and metrics from all the services running on top of Kubernetes.
+
+In addition, one of the Pods in the DaemonSet will constantly hold a _leader lock_ which makes it responsible for
+handling cluster-wide monitoring.
+Find more information about leader election configuration options at <<kubernetes_leaderelection-provider, leader election provider>>.
+This instance is used to retrieve metrics that are unique for the whole
+cluster, such as Kubernetes events or
+https://github.com/kubernetes/kube-state-metrics[kube-state-metrics].
+
+
+Everything is deployed under the `kube-system` namespace by default. To change
+the namespace, modify the manifest file.
+
+To download the manifest file, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-managed-kubernetes.yaml
+------------------------------------------------
+
+[discrete]
+== Settings
+
+{agent} is enrolled to a running {fleet-server} using `FLEET_URL` parameter.
+The `FLEET_ENROLLMENT_TOKEN` parameter is used to connect {agent} to a
+specific {agent} policy.
+To learn how to get an enrollment token from {fleet}, see <<fleet-enrollment-tokens>>.
+
+To specify different destination/credentials,
+change the following parameters in the manifest file:
+
+[source,yaml]
+------------------------------------------------
+- name: FLEET_URL
+  value: "https://fleet-server_url:port"
+- name: FLEET_ENROLLMENT_TOKEN
+  value: "token"
+- name: FLEET_SERVER_POLICY_ID
+  value: "fleet-server-policy" <1>
+- name: KIBANA_HOST
+  value: ""
+- name: KIBANA_FLEET_USERNAME
+  value: ""
+- name: KIBANA_FLEET_PASSWORD
+  value: ""
+------------------------------------------------
+<1> To learn how to create a policy, refer <<create-a-policy-no-ui>>.
+
+
+// Begin collapsed section
+[%collapsible]
+.Configuration details
+====
+****
+
+[cols="2*<a"]
+|===
+| Settings | Description
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-url]
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-enrollment-token]
+
+include::configuration/env/shared-env.asciidoc[tag=fleet-server-policy-id]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-host]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-fleet-username]
+
+include::configuration/env/shared-env.asciidoc[tag=kibana-fleet-password]
+|===
+
+Refer to <<agent-environment-variables>> for all available options.
+
+****
+====
+
+
+[discrete]
+== Deploy
+
+If planning to deploy `state_*` datasets of Kubernetes package,
```

Comment:
I think it can be removed as it was already mentioned. Maybe we should adjust origin doc page as well